File: Python/Program/VGA.py
from Komponen import Komponen

class VGA(Komponen):
    def __init__(self, memory_gb, merk, nama):
        Komponen.__init__(self, merk, nama) 
        self.memory_gb = memory_gb

    def get_memory_gb(self):
        return self.memory_gb

File: C++/Program/VGA.cpp
#pragma once
#include <iostream>
#include <string>
#include "Komponen.cpp"

using namespace std;

class VGA : public Komponen
{
private:
    int memoryGB; 
public:
    VGA()
    {

    } 

    VGA(int memoryGB, string merk, string nama) : Komponen(merk, nama)
    {
        this->memoryGB = memoryGB;
    }

    void setMemoryGB(int memoryGB)
    {
        this->memoryGB = memoryGB;
    }

    int getMemoryGB()
    {
        return memoryGB;
    }

    ~VGA() {}
};